static/js/containers/UserPage_test.js

```diff
@@ -135,6 +135,9 @@ describe("UserPage", function() {
             TestUtils.Simulate.click(save);
             let state = helper.store.getState();
             assert.deepEqual(state.profiles.jane.edit.errors, validationExpectation);
+            let validationInfoText = document.querySelector('.validation-alert').
+              querySelector('.message').textContent;
+            assert.deepEqual(validationInfoText, "Please fix the errors above and try again");
```

Comment:
Not totally sure what you mean, do you mean make the error message a property of the `ValidationAlert` class?